recipe/patch_yaml_model.py

```diff
@@ -196,9 +212,14 @@ class _ThenClauseItem(_ForbidExtra):
         None,
         description="Make a dependency version constraint looser",
     )
+    change_depends: _Name_LowerBound_UpperBound_Other = Field(
+        None,
+        description="Change a dependency version constraint",
+    )
 
 
 class PatchYaml(_ForbidExtra):
+    model_config = ConfigDict(coerce_numbers_to_str=True)
```

Comment:
I'm not sure if this `model_config` assignment overrides the one in the `_ForbidExtra` baseclass.

Reply:
I think you're probably right. This change is not related to the focus of this PR, anyway. I just hit a rather opaque error about schema problems, and the issue was a version like `1.10` being interpreted as a float and triggering the schema violation. Coercing to strings seems like a good idea, but also a subject for another PR.